src/stackwalk.c

```diff
@@ -321,6 +321,7 @@ static void decode_backtrace(jl_bt_element_t *bt_data, size_t bt_size,
 
 JL_DLLEXPORT jl_value_t *jl_get_backtrace(void)
 {
+    JL_TIMING(STACKWALK, STACKWALK_Backtrace);
```

Comment:
These are *so* nice to have.

Would be really nice also to add a signpost for when the error printing gets triggered so that we can easily spot the compilation that also delays `Ctrl-C` interactions, etc. But I'm assuming that entrypoint is on the Julia side?

Reply:
Entry point for these is on the Julia side, but if it's for one specific instance the call graph can be used to identify the caller. I'm thinking this is mostly helpful to identify usage patterns that trigger a lot of backtrace overhead.

Reply:
Yeah, I'm happy just to have this event for now